<?xml version="1.0"?>
<launch>
    <node name="adv_manip_node" pkg="task_planner_pkg" type="advanced_manipulation.py" output="screen">
    </node>
    <node name="end_effectors_node" pkg="task_planner_pkg" type="end_effectors.py" output="screen">
    </node>
    <node name="test_node" pkg="task_planner_pkg" type="test.py" output="screen">
    </node>
</launch>  
